README.md

```diff
@@ -1,70 +1,158 @@
+</ul>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
 
-### Making a Progressive Web App
+### Setup
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+Clone this repository to your desired folder: git@github.com:BranBayou/math-magician.git
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Advanced Configuration
+### Install
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+1. Clone the repo:
+  https://github.com/BranBayou/math-magician
+2. Install npm packages:
+  npm.install
 
-### Deployment
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+### Usage
 
-### `npm run build` fails to minify
+To run tests, run the following command:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+<ul>
+<li>run npm start</li>
+<li>run npm test</li>
+</ul>
+
+### Run Tests
+
+to run tests, run the following command
+
+<ul>
+  <li>npx stylelint "**/*.{css,scss}"</li>
+  <li>npx eslint "**/*.{js,jsx}"</li>
+</ul>
+
+<!-- ### Deployment
+
+This page has been deployed with gitbub pages. -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author**
+
+👤 Berhanu Baye
+- GitHub: [@BranBayou](https://github.com/BranBayou)
+- Twitter: [@bran-baye](https://twitter.com/bran_baye)
+- LinkedIn: [Berhanu Baye](https://www.linkedin.com/in/bran-baye/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Future Features
+
+- Make it a complete project
+- Make user friendly interface
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/BranBayou/math-magician/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project to give a positive feedback and comments that will go along way in making the project even better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for this exercise.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE) licensed.
```

Comment:
- [ ] I would kindly advise you to work on the MIT license link as currently the link is broken and throwing error 404. Having a working link will make your README file looks more descriptive and professional, kindly work on it.

MIT 👇 

<img width="949" alt="image" src="https://user-images.githubusercontent.com/78634841/237010597-206199dc-7a11-4734-b3d3-1b35628ab9d1.png">